packages/expo-updates/android/src/main/java/expo/modules/updates/loader/FileDownloader.java

```diff
@@ -135,7 +143,8 @@ public void onCompleted(boolean isValid) {
                   }
               );
             } else {
-              Manifest manifest = ManifestFactory.getManifest(manifestJson, configuration);
+              preManifest.put("isVerified", false);
+              Manifest manifest = ManifestFactory.getManifest(preManifest, configuration);
```

Comment:
Previously we only set `isVerified` to `false` on unsigned manifests from XDL. `isVerified` was never set on unsigned manifests from non-XDL sources. This seems inconsistent with the explicit setting for XDL.

Following 'explicit is better than implicit' as well as this code comment:
```
// We should treat these manifests as unsigned rather than signed with an invalid signature. 
```	

This PR centralizes the treatment of unsigned manifests from XDL and non-XDL sources to these lines.